### ISSUE TITLE
Remove all explicity logging.setLevel calls

### ISSUE THREAD

**Issue by stscijgbot** [spacetelescope/jwst#4182]
Issue [JP-1090](https://jira.stsci.edu/browse/JP-1090) was created by Jonathan Eisenhamer:

The whole of JWST code is littered with logging.setLevel calls, short-circuiting any usefullness of the logging facility.

Remove them. Just remove them...please...gone...go...squashed...

**Comment by pllim** [comment]
@melanieclarke I don't think this is relevant anymore? Close?

**Comment by melanieclarke** [comment]
Yes, I think we can close this now, after #9644.   It looks like his issue got disconnected from the associated JP ticket.  I will close it too.

We do still need stdatamodels to remove explicit setLevel calls, but I'll file a separate ticket on that repo.
